gtsofa/medium-api
# #166213692 Configure PostgreSQL db engine for the app

### What is this PR supposed to do?
- App uses a postgresql database engine
### How should this be manually tested?
- clone the repo and navigate to the project directory
- check out to  this `ch-configure-medium-clone-to-use-postgresql-db-engine-166213692` branch
- start the server 

### Relevant PT stories
- [166213692](https://www.pivotaltracker.com/n/projects/2350589/stories/166213692)
